properties/service_fabric_common.props

```diff
@@ -22,10 +22,14 @@
 
     <!-- Version for binaries, nuget packages generated from this repo. -->
     <!-- TODO: Versions numbers are changed here manually for now, Integrate this with GitVersion. -->
-    <MajorVersion>9</MajorVersion>
-    <MinorVersion>0</MinorVersion>
-    <BuildVersion>2</BuildVersion>
+    <MajorVersion Condition="'$(MSBuildProjectName)' != 'Microsoft.ServiceFabric.Diagnostics'">9</MajorVersion>
+    <!-- Major Version for Microsoft.ServiceFabric.Diagnostics binary - Bump together with normal MajorVersion -->
+    <!-- We migrated Diagnostics package from WindowsFabric repo and we need to keep the versioning consistent -->
+    <MajorVersion Condition="'$(MSBuildProjectName)' == 'Microsoft.ServiceFabric.Diagnostics'">12</MajorVersion>
```

Comment:
I think this is fine for now. However, we will need to generalize this when we bring the next set of projects over from the WindowsFabric repo. I wonder if we should stop the versioning discrepancy and align the SDK versions to the runtime versions. I can't think of any reason to continue with it. But let's do it as a separate work item.